### PR TITLE
Update recommended syntax; #567

### PR DESCRIPTION
**Pull request by vreuter**
Actually addresses https://github.com/scala/collection-strawman/issues/467

### CODE REVIEW

File: src/library/scala/collection/IterableOnce.scala [hunk 1]
```diff
@@ -43,37 +43,37 @@ final class IterableOnceExtensionMethods[A](private val it: IterableOnce[A]) ext
     case _ => it.iterator.foreach(f)
   }
 
-  @deprecated("Use factory.from(it) instead of it.to(factory) for IterableOnce", "2.13.0")
+  @deprecated("Use it.to(factory) instead of factory.from(it) for IterableOnce", "2.13.0")
```
Comment (julienrf) [inline src/library/scala/collection/IterableOnce.scala:46]: Have you tried that syntax? (`it.to(factory)`)

I suspect that you will get the deprecation warning. I think the syntax should be `it.iterator.to(factory)`.

Comment (vreuter) [reply src/library/scala/collection/IterableOnce.scala:46]: Ah alright, will change this

**Comment by julienrf** [comment]
Thanks @vreuter! Can you please refer to the issue in the commit message by prefixing it with the repository first: scala/collection-strawman#567.

**Comment by vreuter** [comment]
Closing in favor of https://github.com/scala/scala/pull/6822

**Pull request closed without merging**